### DISCUSSION TITLE
Labs problem with white balance in RAW Timelapse

### DISCUSSION THREAD

**Discussion by MrProst** [gopro/labs#350] [General]
When recording a timelapse in RAW format, after about 100-150 frames of recording, the white balance changes and a vignette effect appears.
This happens every time a timelapse is recorded.
![G0025593](https://github.com/gopro/labs/assets/133163316/8cb3425c-8e49-4705-8ac0-43c26cfa8c35)
![G0025592](https://github.com/gopro/labs/assets/133163316/ee06e96c-26f0-4229-abbf-f26470d58be0)

**Comment by dnewman-gpsw** [comment]
Yes, that is normal for Raw.  JPEG has all the processing, Raw does not.  White balance in Raw is just metadata, it is up to the Raw development tool to do the white balance corrections.   The flat fielding (corner vignetting correction) is also a postprocess.

**Reply by MrProst** [reply]
> Yes, that is normal for Raw. JPEG has all the processing, Raw does not. White balance in Raw is just metadata, it is up to the Raw development tool to do the white balance corrections. The flat fielding (corner vignetting correction) is also a postprocess.

This is all great, but in the end it turns out that after shooting I get a timelapse where the white balance is different. Why does the metadata change during shooting if I have locked the white balance?

**Reply by dnewman-gpsw** [reply]
Can you confirm the metadata changed?   That would be a bug if it did under locked WB.

**Reply by MrProst** [reply]
> Can you confirm the metadata changed? That would be a bug if it did under locked WB.

This happens with white balance locked.

**Reply by dnewman-gpsw** [reply]
Can you confirm this is metadata issue?  Does LightRoom show the white balance changing in the data.  This is not a Labs issue, so the issue is more core, if the lock in applied to raw images.

**Reply by MrProst** [reply]
> Can you confirm this is metadata issue? Does LightRoom show the white balance changing in the data. This is not a Labs issue, so the issue is more core, if the lock in applied to raw images.

[G0028172.zip](https://github.com/gopro/labs/files/11466603/G0028172.zip)
[G0028173.zip](https://github.com/gopro/labs/files/11466604/G0028173.zip)
![2](https://github.com/gopro/labs/assets/133163316/1b096233-0f10-49a3-af6b-34bc8a5d942d)
![1](https://github.com/gopro/labs/assets/133163316/d574ad72-c5d3-40ca-8ae9-c8c20f2ddce6)
I don't know Adobe Lightroom very well, so maybe my screenshots won't help. Therefore, I am attaching two files in DNG format, packed in zip. These are two consecutive frames, but there has been a change in white balance between them. I set the white balance to "natural", it switched to "auto" without my participation.
